programmers/난이도별/level01.최대공약수와_최소공배수/dkdlelk99.py

```diff
@@ -0,0 +1,10 @@
+def solution(n, m):
+    a, b = max(n,m), min(n,m)
```

Comment:
```suggestion
    a = min(n,m)
```
어차피 max는 안쓰이는 것 같아요~